infrastructure/Program.fs

```diff
@@ -15,14 +15,15 @@ let rg = $"{baseName}-rg"
 let myWeb = webApp {
     name webAppName
     system_identity
-    setting "storage-account-name" storageAccountName
+    operating_system OS.Windows
+    setting "StorageAccountName" storageAccountName
```

Comment:
I've almost always seen it as PascalCase in .NET land. And often they're mapped directly to objects with PascalCase properties so that makes sense to me. It's the casing I always use.